src/lib/markbind/src/parser.js

```diff
@@ -581,53 +570,52 @@ class Parser {
     });
   }
 
-  _rebaseReference(node, foundBase) {
+  /**
+   * Pre-renders the baseUrl of the provided node and its children according to
+   * the site they belong to, such that the final call to render baseUrl correctly
+   * resolves baseUrl according to the said site.
+   */
+  _rebaseReference(node) {
     if (_.isArray(node)) {
-      return node.map(el => this._rebaseReference(el, foundBase));
+      return node.map(el => this._rebaseReference(el));
     }
     if (Parser.isText(node)) {
       return node;
     }
-    // Rebase children element
-    const childrenBase = {};
-    node.children.forEach((el) => {
-      this._rebaseReference(el, childrenBase);
-    });
-    // rebase current element
-    if (node.attribs[ATTRIB_INCLUDE_PATH]) {
-      const filePath = node.attribs[ATTRIB_INCLUDE_PATH];
-      let newBaseUrl = urlUtils.calculateNewBaseUrls(filePath, this.rootPath, this.baseUrlMap);
-      if (newBaseUrl) {
-        const { relative, parent } = newBaseUrl;
-        // eslint-disable-next-line no-param-reassign
-        foundBase[parent] = relative;
-      }
-      // override with parent's base
-      const combinedBases = { ...childrenBase, ...foundBase };
```

Comment:
This seems to be the commit that added the `foundBase` change:
https://github.com/MarkBind/markbind/commit/0faa1b731a9d4e4675b4f88d36f411f54f2a7580

Reply:
Thanks for this! Unfortunately the PR dosen't really reveal the workings of `combinedBases`, but

> What is the rationale for this request?
> When including files belonging to an inner website, the computation of baseUrl is wrong if the depth of the inclusion is only 1.

Going by this, it should be sufficient to resolve the baseurl only when the include's file and included file belongs to different sites. The likely cause of the issue was wrongly resolving against `ATTRIB_CWF` instead of `ATTRIB_INCLUDE_PATH`, and perhaps the above mentioned irregularities with `foundBase`.

I was looking at https://github.com/MarkBind/markbind/commit/64b2ff6d93a457869fe965d896995dd7d1d8083e previously, where `foundBase` was included with the function right from the start. 
It dosen't reveal much as well unfortunately, perhaps @Gisonrg could comment!